db/index.js

```diff
@@ -0,0 +1,56 @@
+const { Client } = require('pg')
+const client = new Client({
+  database: 'todolist'
+})
+
+client.connect()
+
+const addToDatabase = (task) => {
+  client.query('INSERT INTO todolist (task) VALUES ($1) RETURNING id', [task])
+    .then(response => {
+      console.log(`Created task ${response.rows[0].id}`)
+      client.end()
+    })
+    .catch(err => console.error(err))
+}
+
+const listAllFromDB = () => {
+  client.query('SELECT * FROM todolist')
+    .then(result => {
+      const widestIDString = Math.max.apply(null, result.rows.map(task => task.id.toString().length))
```

Comment:
Does this determine how wide the widest item is so you know how many "-"'s to make in the shell? If yes this const name makes sense 👍  If it's for something else I wonder why it's called widest.